meeting/meeting_may_10_2022.md

```diff
@@ -0,0 +1,51 @@
+# May 10th (6 PM CST)/11th (9 AM JST) Meeting 2022
+
+[A Telegram channel for people who ask you about where to join a channel](https://t.me/BitcoinDLCs)
+
+## Individual Updates (Sync)
+
+* **#Query(Nadav)**
+* **#Query(Chris)**
+* **#Query(Matt)**
+* **#Query(Tibo)**
+* **#Query(Ben)**
+* **#Query(Lloyd)**
+* **#Query(Antoine)**
+* **#Query(Aki)**
+* **#Query(Philipp)**
+* **#Query(all)**
+
+## Implementation Updates (Sync)
+
+* bitcoin-s
+
+* rust-dlc
+
+* atomic finance
+
+* gun.fun
+
+* others?
+
+## Specification Writing
```

Comment:
This one is still pending as well: https://github.com/discreetlogcontracts/dlcspecs/pull/186